src/clr-addons/history/history.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2018-2020 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component, Input, OnInit } from '@angular/core';
+import { ClrHistoryModel } from './history-model.interface';
+import { ClrHistoryService } from './history.service';
+
+@Component({
+  selector: 'clr-history',
+  templateUrl: './history.html',
+})
+export class ClrHistory implements OnInit {
+  @Input('username') username: string;
```

Comment:
we follow clarity styleguides, so input variables should prefix 'clr' (clrUsername in this case)
https://github.com/vmware/clarity/blob/master/docs/CODING_GUIDELINES_ANGULAR.md